mailpoet/views/subscription/unsubscribe_reason.html

```
@@ -0,0 +1,43 @@
+<% block content %>
+<form class="mailpoet-unsubscribe-reason" action="<%= actionUrl|escape %>" method="post">
+  <input type="hidden" name="_wpnonce" value="<%= nonce|escape('html_attr') %>">
+  <fieldset>
+    <legend><%= __('Please let us know why you unsubscribed:', 'mailpoet') %></legend>
+    <% for reason, label in reasons %>
+      <p>
+        <label>
+          <input type="radio" name="reason" value="<%= reason|escape('html_attr') %>" required>
+          <%= label %>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)